frontend/src/components/LineView.js

```diff
@@ -118,7 +118,7 @@ const LineView = ( {
       <GlobalHotKeys keyMap={keyMap} handlers={handlers} allowChanges>
         <div className="header">
           <div className="left buttons">
-            <LinkButton className="button" icon="level-up-alt" to={sourceViewUrl} />
+            <LinkButton className="button" icon="level-up-alt" to={sourceViewUrl} data-cy="go-to-home-button" />
```

Comment:
Is `data-cy` the best way to do this? It pollutes our source code. I'd assume/prefer that it's possible to grab the element by way of className or scannning for text

Reply:
We can but according to their [docs](https://docs.cypress.io/guides/references/best-practices.html#How-It-Works) it is a good practice that we use `data-cy` to select if we don't have other way.